### DISCUSSION TITLE
allow one ride station to have two entrances and two exits

### DISCUSSION THREAD

**Discussion by 733737** [OpenRCT2/OpenRCT2#17740] [Feature requests]
It would be nice if you could have a station platform with 2 exits and 2 entrances, instead of restricting each station platform to one exit and one entrance. Guests could load faster, and they could get on from two different pathing sections of the park or exit to two different areas of park that aren't connected even though they are relatively close

**Comment by Gymnasiast** [comment]
Can you please stop spamming? You created dozens of topic within a few hours.

**Comment by TheClaudeQC** [comment]
Could be a nice inplementation for multi-dimention coasters (of course, not breaking old ones, like X from magic mountain)
